### ISSUE TITLE
Sometimes once a move is confirmed after block insertion the focus moves to elsewhere so the user cannot keyboard navigate afterwards.

### ISSUE THREAD

**Issue by sappm01** [google/blockly#8972]
### Check for duplicates

- [ ] Sometimes once a move is confirmed after block insertion the focus moves to elsewhere so the user cannot keyboard navigate afterwards.

### Description

Sometimes once a move is confirmed after block insertion the focus moves to elsewhere so the user cannot keyboard navigate afterwards.
Repro steps:
Load page.
Tab to open flyout.
Select a block to insert. The block will heuristically insert to the workspace.
Move the block somewhere else.
Hit “ENTER” to confirm and try to keyboard navigate.
You can also repro the issue if you select a parent block with a value block, and then try to insert a new value block from the flyout to replace the original value block so that the new value block displaces the old one.
Ben: I think this is the same as the 'Move' issue mentioned above.

### Reproduction steps

1. 
2. 
3.

### Stack trace

```shell

```

**Comment by sappm01** [comment]
Delete - Issue is not needed for this.
